doc/dependencies.md

```diff
@@ -17,33 +17,33 @@ You can find installation instructions in the `build-*.md` file for your platfor
 
 | Dependency | Version used | Minimum required | Runtime |
 | --- | --- | --- | --- |
-| [Boost](https://www.boost.org/users/download/) | 1.77.0 | [1.64.0](https://github.com/bitcoin/bitcoin/pull/22320) | No |
-| [libevent](https://github.com/libevent/libevent/releases) | 2.1.12-stable | [2.1.8](https://github.com/bitcoin/bitcoin/pull/24681) | No |
+| [Boost](../depends/packages/boost.mk) | [link](https://www.boost.org/users/download/) | [1.77.0](https://github.com/bitcoin/bitcoin/pull/24383) | [1.64.0](https://github.com/bitcoin/bitcoin/pull/22320) | No |
```

Comment:
I think something went wrong; versions ends up in the "runtime" column here.

Reply:
Fixed in 128e4ed02c03ce8c4ce206f5858dd787d06e1a18